samcli/lib/bootstrap/companion_stack/companion_stack_builder.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
I wonder if we should move the s3 bucket stack to be of this format as well, makes it more consistent experience for us to reason about.

Reply:
Updated S3 template to be using dict.